src/app/product/page.tsx

```diff
@@ -0,0 +1,17 @@
+"use client"
+import React from "react";
+import ProductForm from '../components/productForm'; 
+import { IProduct } from "../products/productInterface";
+import productSevice from "../services/productSevice";
+ 
+
+  function CreateProductItem(product:IProduct) {   
+    productSevice.create(product); 
+    window.history.pushState({}, "", "/products")
```

Comment:
This can be done with a server component and useRouter - router.push('url'). There's no user interactivity on this page to make it a client component.